packages/mobile/src/screens/sign-up-screen/screens/SelectArtistsScreen.tsx

```diff
@@ -41,6 +85,51 @@ export const SelectArtistsScreen = () => {
           )
         })}
       </View>
+      <Formik initialValues={initialValues} onSubmit={handleSubmit}>
+        {({ values, setValues, handleSubmit }) => {
+          const { artists: selectedArtists } = values
+          return (
+            <View>
+              <View>
+                {isLoading
+                  ? null
+                  : artists?.map((user) => {
```

Comment:
Maybe split this inner function into a helper function?

Reply:
thinking ill just create a new component for it :)